spec/presenters/edition_diff_presenter_spec.rb

```diff
@@ -0,0 +1,140 @@
+        link_type: "organisations"
+      )
+    ]
+  end
+
+  let(:edition) do
+    double(:edition,
+      analytics_identifier: nil,
+      base_path: "/path",
+      content_store: "draft",
+      description: "A description",
+      details: {},
+      document_type: "answer",
+      first_published_at: timestamp,
+      last_edited_at: timestamp,
+      need_ids: [],
+      phase: "live",
+      public_updated_at: timestamp,
+      publishing_app: "publisher",
+      redirects: [],
+      rendering_app: "government-frontend",
+      routes: [{ "path" => "/path", "type" => "prefix" }],
+      schema_name: "answer",
+      state: "draft",
+      title: "A title",
+      user_facing_version: 1,
+      update_type: "major",
+      links: organisation_links,
+      change_note: change_note
+    )
+  end
+
+  describe "#call" do
+    context "with links and change_note" do
+      it "returns an edition hash presented for diffing" do
+        expect(subject.call(edition)).to match a_hash_including(
+        "analytics_identifier" => nil,
+        "base_path" => "/path",
+        "content_store" => "draft",
+        "description" => "A description",
+        "details" => {},
+        "document_type" => "answer",
+        "first_published_at" => timestamp,
+        "last_edited_at" => timestamp,
+        "need_ids" => [],
+        "phase" => "live",
+        "public_updated_at" => timestamp,
+        "publishing_app" => "publisher",
+        "redirects" => [],
+        "rendering_app" => "government-frontend",
+        "routes" => [{ "path" => "/path", "type" => "prefix" }],
+        "schema_name" => "answer",
+        "state" => "draft",
+        "title" => "A title",
+        "user_facing_version" => 1,
+        "update_type" => "major",
+        "links" => { "organisations" => [content_id] },
+        "change_note" => change_note.note
+        )
+      end
+    end
+
+    context "without links" do
+      it "returns an edition hash presented for diffing" do
+        allow(edition).to receive(:links).and_return nil
+        expect(subject.call(edition)).to match a_hash_including(
+          "analytics_identifier" => nil,
+          "base_path" => "/path",
+          "content_store" => "draft",
+          "description" => "A description",
+          "details" => {},
+          "document_type" => "answer",
+          "first_published_at" => timestamp,
+          "last_edited_at" => timestamp,
+          "need_ids" => [],
+          "phase" => "live",
+          "public_updated_at" => timestamp,
+          "publishing_app" => "publisher",
+          "redirects" => [],
+          "rendering_app" => "government-frontend",
+          "routes" => [{ "path" => "/path", "type" => "prefix" }],
+          "schema_name" => "answer",
+          "state" => "draft",
+          "title" => "A title",
+          "user_facing_version" => 1,
+          "update_type" => "major",
+          "links" => {},
+          "change_note" => change_note.note
+        )
```

Comment:
so you probably only need to test that this is `a_hash_including("links" => {})` for this context